packages/react-components/react-accordion/src/components/AccordionHeader/useAccordionHeaderStyles.styles.ts

```diff
@@ -20,17 +20,17 @@ const useStyles = makeStyles({
     fontFamily: 'inherit',
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31449&commitId=fcc73e07ac5243f5f7872a6dbd087d485137c829&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>Calendar Compat</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Calendar Compat.Default - RTL.default.chromium.png|652| Changed|
|Calendar Compat.Custom icons.default.chromium.png|644| Changed|
|Calendar Compat.Default.default.chromium.png|652| Changed|

</details>


  <details><summary><b>DatePicker Compat</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.default - High Contrast.opened.chromium.png|1238| Changed|
|DatePicker Compat.default - Dark Mode.opened.chromium.png|565| Changed|
|DatePicker Compat.DateRange- month - Dark Mode.default.chromium.png|565| Changed|
|DatePicker Compat.DateRange- month - High Contrast.default.chromium.png|1238| Changed|
|DatePicker Compat.DateRange- day.default.chromium.png|652| Changed|
|DatePicker Compat.DateRange- week.default.chromium.png|652| Changed|
|DatePicker Compat.showWeekNumbers - High Contrast.default.chromium.png|1238| Changed|
|DatePicker Compat.showWeekNumbers - Dark Mode.default.chromium.png|565| Changed|
|DatePicker Compat.marked dates - High Contrast.default.chromium.png|1238| Changed|
|DatePicker Compat.DateRange- work week.default.chromium.png|652| Changed|
|DatePicker Compat.default.opened.chromium.png|652| Changed|
|DatePicker Compat.DateRange- month.default.chromium.png|652| Changed|
|DatePicker Compat.marked dates - Dark Mode.default.chromium.png|565| Changed|
|DatePicker Compat.showWeekNumbers.default.chromium.png|652| Changed|
|DatePicker Compat.marked dates.default.chromium.png|652| Changed|
|DatePicker Compat.when rendering inline, it should not render behind relatively positioned elements.default.chromium.png|652| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>